tool/mustachio/codegen_aot_compiler.dart

```diff
@@ -116,27 +119,23 @@ Future<Map<_AotCompiler, String>> _deduplicateRenderers(
       continue;
     }
     var firstCompiler = compilers.first;
```

Comment:
This part of the codegen is kinda confusing. Why do we use the first compiler's `_syntaxTree` and `_buildData` when making the `lubCompiler`? Also, perhaps move this variable down there, since it's not being used for `contextStacksLength` anymore

All the other changes seem okay to me.

Reply:
I see! Okay, that makes a lot of sense. Yeah, some extra documentation on that would be great :)